pkg/k8s/service.go

```diff
@@ -0,0 +1,45 @@
+package k8s
+
+import (
+	"github.com/maistra/istio-workspace/pkg/model"
+
+	corev1 "k8s.io/api/core/v1"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	// ServiceKind is the k8s Kind for a Service
+	ServiceKind = "Service"
+)
+
+var _ model.Locator = ServiceLocator
+
+// ServiceLocator attempts to locate the Services for the target Deployment/DeploymentConfig
+func ServiceLocator(ctx model.SessionContext, ref *model.Ref) bool { //nolint[:hugeParam]
```

Comment:
I see `//nolint[:hugeParam]` in quite a few places and I'm wondering if we should perhaps disable this check if it really does not make sense or maybe relax if possible. WDYT?

Reply:
Been wondering the same. Other PR?

Reply:
Sure, but I wonder why we are disabling it in the first place.

Reply:
To be honest I don't really get the check. Everything I read about pointer vs value in go is; don't bother with it for performance, it's a hint at the usage of the object. And that people that really optimize a program avoid pointers because they force heap, values go on the stack. But that might be for smaller objects. 

I would find it a bit odd that you can't just loop over the standard k8s list object if it's so so terrible... _shrug_

Reply:
Ok I will open pr disabling it right now.